src/docker/Dockerfile

```diff
@@ -52,43 +52,31 @@ RUN apt-get update && apt-get install -y \
     automake autoconf
 
 # Git 2.0
-RUN apt-get install -y software-properties-common python-software-properties
+RUN apt-get update && apt-get install -y \
```

Comment:
@mkordas I looked now into it... They already seem well grouped to me? Each tool/stack of tools with it's own, commented section and locked versions. 

[This](https://github.com/yegor256/dockers/blob/master/xjm/Dockerfile) example is much shorter because everything is messed up together

I think it's much readable the way it is now. You see everything nicely grouped rather than trying to read between the lines to figure out that eveything there does and if there is any relation between them.

Still want to regroup?